sdk/schemaregistry/azure-data-schemaregistry/CHANGELOG.md

```diff
@@ -1,17 +1,17 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.3 (2022-03-15)
 
 ### Features Added
 
 - Added interfaces from `com.azure.core.client.traits` to `SchemaRegistryClientBuilder`.
 - Added `retryOptions` to `SchemaRegistryClientBuilder`.
 
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+#### Dependency Updates
 
-### Other Changes
+- Update `azure-core` dependency to `1.26.0`.
```

Comment:
We should also include the netty version update here.

Reply:
privacy issues data loss #fraud